### DISCUSSION TITLE
use rslib for making a svelte library

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [web-infra-dev/rslib#205] [Q&A]
hi, i wan to use rslib for making a svelte library , as  if i use rsbuild svelte plugin, it would compile it to js, which i don't want , i just wana preprocess the svelte file and emit it and its types .d.ts file, how to do that?

**Comment by Timeless0911** [comment]
There is no special support yet, we will consider related requirements after releasing a stable version.
